heuristic/classes/Route.py

```diff
@@ -51,3 +52,22 @@ def _plan_cost(self, problem: Problem) -> float:
             cost += Stacks.cost(customer, problem, before, after)
 
         return cost
+
+    def remove_customer(self, customer: int):
+        """
+        Removes customer from route and items with the customer as destination
+        or origin from the loading plan.
+        """
+        idx = self.customers.index(customer)
+        delivery_item = Item(Problem.demands[customer], DEPOT, customer)
+        pickup_item = Item(Problem.demands[customer], customer, DEPOT)
+
+        for stacks in self.plan[:(idx - 1)]:
+            stack = stacks.find_stack(delivery_item)
+            stack.stack.remove(delivery_item)
```

Comment:
This is not sufficient to remove the item - it also needs to be removed from the set member. Write a `remove` wrapper function on `Stack`.